apps/service/data/assembly_zips.go

```diff
@@ -0,0 +1,8028 @@
+// Code generated from 'Assembly Districts by Zip - Raw.csv'; DO NOT EDIT.
+
+package data
+
+type Zip string
+
+// ZipDistrictPercent maps ZIP code -> district number -> percent of population with that district in the zipcode (0-100).
```

Comment:
"percent of population with that district in the zipcode" seems possibly ambiguous
how about "percent of the zip code's population that is in that district" ?